packages/client/src/pages/common/nav/nav.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { AppBar, Toolbar, Typography, Button,ButtonBase } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import ArrowDropDownIcon from "@material-ui/icons/ArrowDropDown";
+
+const useStyles = makeStyles((theme: any) => ({
+  root: {
+    flexGrow: 1,
+  },
+  menuButton: {
+    marginLeft: theme.spacing(2),
+  },
+  iconContainer: {
+    flexGrow: 1,
+  },
+  profileButton: {
+    textTransform: "none",
+  }
+}));
+
+const NavBar = () => {
+  const classes = useStyles();
+
+  return (
+    <AppBar position="static" color="primary">
```

Comment:
Are we always gonna show appbar?

Reply:
Hmm have we decided if it's going to be sticky or if we're going to have a "go to top" button? I can change that later if we reach a conclusion

Reply:
if

Reply:
else